src/test/java/com/puppycrawl/tools/checkstyle/AbstractAutomaticBeanTest.java

```diff
@@ -278,6 +280,46 @@ public void testBeanConvertersUri3() {
         }
     }
 
+    @Test
+    public void testBeanConverterPatternArray1() throws Exception {
+        final ConverterBean bean = new ConverterBean();
+        final DefaultConfiguration config = new DefaultConfiguration("bean");
+        final String patternString = "^a*$,^b*$, ^c*$";
+        final List<String> expectedPatternString = Arrays.asList("^a*$", "^b*$", " ^c*$");
+        config.addProperty("patterns", patternString);
+        bean.configure(config);
+
+        assertWithMessage("invalid result")
+                .that(bean.patterns)
+                .hasLength(3);
+        assertWithMessage("invalid result")
+                .that(expectedPatternString)
+                .containsExactlyElementsIn(
+                        Arrays.stream(bean.patterns)
+                                .map(Pattern::pattern)
+                                .collect(Collectors.toUnmodifiableList()));
+    }
+
+    @Test
+    public void testBeanConverterPatternArray2() throws Exception {
```

Comment:
`testBeanConverterPatternArraySingleElement`

and please do make test for empty value `""` for list of patterns.

Reply:
done